### DISCUSSION TITLE
部署一直失败，升级了$5服务还是这样，快粉转黑了【抓狂】

### DISCUSSION THREAD

**Discussion by xupipi1125** [zeabur/.github#10] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/109848250/234240211-006e15b9-98ef-441b-95c1-a12cccffe3bd.png)

**Comment by yuaanlin** [comment]
請問您的服務在本地能 build 成功嗎？

**Comment by MichaelYuhe** [comment, marked as answer]
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/63531512/234261374-1438ee03-950c-4e4c-bca5-c21361dbb109.png">
本地构建也失败了，您可能需要查看下代码或者 notion 设置，有问题可以加入 Discord 群组联系我们活重新打开讨论～
